README.md

```diff
@@ -1,2 +1,35 @@
 # ui-plugin-find-eresource
-ERM-eresource-finder plugin for Stripes
+
+Copyright (C) 2017-2019 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License,
+Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
+
+## Introduction
+
+This package furnishes a single Stripes plugin of type `find-eresource`,
+which can be included in Stripes modules by means of a `<Pluggable
+type="find-eresource">` element. See [the *Plugins*
+section](https://github.com/folio-org/stripes-core/blob/master/doc/dev-guide.md#plugins)
+of the Module Developer's Guide.
+
+## Props
+
+| Name | Type | Description | Required | default
```

Comment:
This table isn't formatted properly - it doesn't render properly.

Reply:
Updated